### DISCUSSION TITLE
Save certs in named folders

### DISCUSSION THREAD

**Discussion by akapsch** [NginxProxyManager/nginx-proxy-manager#2673] [Ideas]
Hi. I have my /etc/letsencrypt container folder bound to my hosts /etc/letsencrypt folder. I do this for other containers as well so I can have one container retrieving and updating certs. Previously I did this with SWAG, but I figured it could be done with NPM. I noticed however that NPM saves the certs in folders such as "NPM-02" instead of "example.com" or "*.example.com". It would be cool to have a checkbox / field in NPM when creating a certificate that would let us specify the name of the folder it stores the cert in. That way other containers can find the cert easier.

**Comment by idef1x** [comment]
Totally agree...I need to bind mount the letsencrypt folder into my mailservers /etc/letsencrypt, which needs the domain name as a folder....
